src/lib/es5.d.ts

```diff
@@ -1245,8 +1249,7 @@ interface ReadonlyArray<T> {
      * @param callbackfn A function that accepts up to four arguments. The reduce method calls the callbackfn function one time for each element in the array.
      * @param initialValue If initialValue is specified, it is used as the initial value to start the accumulation. The first call to the callbackfn function provides this value as an argument instead of an array value.
      */
-    reduce(callbackfn: (previousValue: T, currentValue: T, currentIndex: number, array: readonly T[]) => T): T;
-    reduce(callbackfn: (previousValue: T, currentValue: T, currentIndex: number, array: readonly T[]) => T, initialValue: T): T;
+    reduce(callbackfn: (previousValue: T, currentValue: T, currentIndex: number, array: readonly T[]) => T, initialValue?: T): T;

```

Comment:
I can't imagine how this will be a breaking change, but removing/adding overloads nearly always is somehow

Reply:
BTW is #7014 not resolved also due to compatibility? It has been an issue for years.